### PR TITLE
Refactor header of web UI

### LINKED ISSUE #287: ui/templates: refactor headers
headers are copy pasted in the 3 templates which isn't clean
(example is #286 )

### PR DESCRIPTION
**Pull request by thekuwayama**
This PR refactors web UI. There are same html so

- Add ui/templates/header.html
- Replace same html ui/templates/header.html using template

Fixes #287

### CODE REVIEW

**Review by ldemailly — APPROVED** [review]
Thank you - you have tested that it still works on multiple pages ?

**Comment by ldemailly** [comment]
@thekuwayama Thanks a lot for your contribution. In order to merge this, may I ask you to open a dummy PR in istio/istio in order to sign the CLA there ? (the cla bot unfortunately isn't setup in this repo)
(to attribute your work to the Istio/Fortio Authors)

**Comment by ldemailly** [comment]
example of what I mean/is needed: https://github.com/istio/istio/pull/9156

**Comment by thekuwayama** [comment]
@ldemailly Thanks for PR example !!

PR: https://github.com/istio/istio/pull/11501

**Comment by thekuwayama** [comment]
> Thank you - you have tested that it still works on multiple pages ?

@ldemailly Yes, I have tested it. 😄

**Comment by thekuwayama** [comment]
@ldemailly I tried add check to Webtest.sh. 
Could you comment if this change is unnecessary (or anything)?

**Review by ldemailly — APPROVED** [review]
thanks for the addition of the test

File: Webtest.sh [hunk 1]
```diff
@@ -72,8 +71,19 @@ if [ "$SIZE" -lt 8191 ] || [ "$SIZE" -gt 8400 ]; then
...
 
-# Check the main page
-$CURL $BASE_FORTIO
+# Check main, sync, browse pages
+VERSION=$(docker exec $DOCKERNAME $FORTIO_BIN_PATH version -s)
+for p in "" browse sync; do
+  # Check the page doesn't 404s
+  $CURL ${BASE_FORTIO}${p}
+  # Check that page includes 3 logos
+  LOGOS_COUNT=$($CURL ${BASE_FORTIO}${p} | grep "${VERSION}/static/img/logo.svg" | wc -l)
```
[the remark this answers is not in the document]
Comment (thekuwayama) [reply Webtest.sh]: https://github.com/fortio/fortio/pull/319/commits/3277c4a54e8945ce8e84afd5b917c8cc6cd04763 👍

**Comment by thekuwayama** [comment]
Thanks!

**Pull request merged**